human_position/Keypoint[] keypoints
int32 x
int32 y
int32 w
int32 h
float32 waving_score